applications/nrf5340_audio/src/modules/led.c

```diff
@@ -224,15 +275,20 @@ static int led_set(uint8_t led_unit, enum led_color color, bool blink)
 		return ret;
 	}
 
-	led_units[led_unit].user_cfg.blink = blink;
-	led_units[led_unit].user_cfg.color = color;
+	led_units_map[led_unit]->user_cfg.blink = blink;
+	led_units_map[led_unit]->user_cfg.color = color;
 
 	return 0;
 }
 
-int led_on(uint8_t led_unit, ...)
+int led_on(enum led_unit_assignment led_unit, ...)
 {
-	if (led_units[led_unit].unit_type == LED_MONOCHROME) {
+	if (led_units_map[led_unit] == NULL) {
+		LOG_WRN("LED unit %d is not assigned to an LED(s)", led_unit);
```

Comment:
Should we need a WRN here? If the LED does not exist on that DK, IMO that should be a INF.